src/models/map.ts

```diff
@@ -0,0 +1,22 @@
+import { LatLngExpression, Point, Map} from "leaflet";
+import {action, observable} from "mobx";
+
+export class MapModel {
+  @observable public east = 180;
+  @observable public north = 90;
+  @observable public west = -190;
```

Comment:
Should this be -180?

Reply:
Thanks, yes, -180. I'll fix it in one of the upcoming PRs (already did it in my local code).